express-notes-api/server.js

```diff
@@ -0,0 +1,152 @@
+// imports
+import express from 'express';
+import { readFile, writeFile } from 'node:fs/promises';
+
+// Global variables
+const app = express();
+const filePath = 'data.json';
+
+// reads data from data.json
+const data = await readFile(filePath, 'utf8');
+const jsonData = JSON.parse(data);
+
+// applies middleware to parse json data in request bodies
+app.use(express.json());
+
+// GETS a list of all notes from data.json
+app.get('/api/notes', async (req, res) => {
+  try {
+    res.json(jsonData);
```

Comment:
This is close but not quite right since data.json has both a notes field and a nextId field. You could use `res.json(jsonData.notes)` but that wouldn't be quite right, either, since the goal is to return the notes in an array and jsonData.notes is an object. So you need to loop through the notes and push them into an array.